code/modules/mob/living/carbon/xenomorph/Embryo.dm

```diff
@@ -218,6 +218,18 @@
 		to_chat(new_xeno, "Talk in Hivemind using <strong>;</strong> (e.g. ';My life for the queen!')")
 		playsound_client(new_xeno.client, 'sound/effects/xeno_newlarva.ogg', 25, 1)
 
+	// Inform observers to grab some popcorn if it isnt nested
+	if(!HAS_TRAIT(affected_mob, TRAIT_NESTED))
```

Comment:
This all should be tabbed over to be underneath the picked check, else this will be spammed whenever there isn't a candidate as it repeats to find one. It should only happen when there is a candidate.

Reply:
Sorry it looks like its now setting stage to 6; last I remember it didn't here so no spamming. Regardless though theres no popcorn to be had if the larva just burrows because theres no one.

Reply:
True, this works out anyway.

Reply:
So actually, feel free to undo that indentation. So if we actually make it to stage 6 without a candidate the dchat message can help someone to actually take over that larva via join as xeno

Alternatively, keep it indented and tinker with having that call repeat during phase 6 if we have no mind yet. I swore it repeated the call but not seeing atm how it would repeat try to get a candidate.

Reply:
Alright, will change back